dbpedia1.js

```diff
@@ -0,0 +1,171 @@
+/////////////////////EXTRACTS 21st,20th,19th,18th and 17th Century classical musicians///////////////
+var fs = require('fs');
+var sleep = require('system-sleep');
+var request = require("request");
+var cheerio = require('cheerio');
+var jsesc = require('jsesc');
+var source_link = [],
+    url = [],
+    musicians = [];
+url.push("http://dbpedia.org/page/Category:21st-century_classical_musicians");
+url.push("http://dbpedia.org/page/Category:20th-century_classical_musicians");
+url.push("http://dbpedia.org/page/Category:19th-century_classical_musicians");
+url.push("http://dbpedia.org/page/Category:18th-century_classical_musicians");
+url.push("http://dbpedia.org/page/Category:17th-century_classical_musicians");
```

Comment:
Any reason we are not including http://dbpedia.org/page/Category:16th-century_classical_musicians ?